app/main.py

```diff
@@ -1 +1,24 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command = command.split()
```

Comment:
I think it would be much better to create variables for both command arguments, since it would be easier to follow your code logic with named variables than with `command[0/1/2]`.

Reply:
make sense